solcOptimiserSettings.js

```diff
@@ -1,4 +1,4 @@
 module.exports = {
     enabled: true,
-    runs: 430
+    runs: 30000
```

Comment:
can't go higher??

Reply:
It's possible, but I think orderbook or eth2dai reserve will exceed max bytecode size, causing tests to fail